src/styles/blocks/main.scss

```diff
@@ -0,0 +1,28 @@
+main {
+  padding: 50px 40px;
+
+  --column-counts: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-counts), 200px);
```

Comment:
When using CSS custom properties, ensure they are declared before being used. The declaration of '--column-counts' should come before its usage in 'grid-template-columns'.